### ISSUE TITLE
Incorrect documentation of load_kernel() spice function on the TudatPy API website

### ISSUE THREAD

**Issue by sbcowan** [tudat-team/tudat-multidoc#34]
I was reading through some of the spice functions looking for load_standard_kernels(). I read the documentation for load_kernel(), which states that the function _cleares_ all loaded kernels, but it should load a specific kernel, I imagine. The documentation is identical to the clear_kernels() function, where the description does fit the function name.

**Comment by DominicDirkx** [comment]
Hi Sean, thanks for pointing this out! @gaffarelj, next time you make an update of the API docs, could you work this fix in as well?

**Comment by gaffarelj** [comment]
Turns out the docstring was correct in `tudat-multidoc`, but the `clear_kernels()` doctrings was incorrectly loaded for the `load_kernel()` function. Good catch.

I just pushed a fix to this ([b5cde5d](https://github.com/tudat-team/tudatpy/commit/b5cde5d8de58f8c106c3920bb49ef98147a74abe)). We will however have to wait for a little bit before seeing the changes on the API docs website (a new `tudat` conda package has to be built first to incorporate a new little addition).
